README.md

```diff
@@ -159,3 +159,38 @@ spec:
         - "--dns01-recursive-nameservers=1.1.1.1:53"
         - "--dns01-recursive-nameservers-only"
 ```
+## Using cert-manager metrics and monitoring
+
+Cert-Manager supports metrics reporting using Prometheus in Openshift. 
+
+Please follow the steps to enable the monitoring for user-defined projects in Openshift ([Doc link](https://docs.openshift.com/container-platform/4.10/openshift_images/managing_images/using-image-pull-secrets.html#images-allow-pods-to-reference-images-from-secure-registries_using-image-pull-secrets)):
+
+Please follow the steps below to generate the metrics of the cert-manager controller scraped by openshift cluster monitoring.
+
+Apply the service-monitor.yaml
+
+```apiVersion: monitoring.coreos.com/v1
+kind: ServiceMonitor
+metadata:
+  labels:
+    app: cert-manager
+    app.kubernetes.io/component: controller
+    app.kubernetes.io/instance: cert-manager
+    app.kubernetes.io/name: cert-manager
+    app.kubernetes.io/version: v1.9.1
+  name: cert-manager
+  namespace: cert-manager
+spec:
+  endpoints:
+  - interval: 30s
+    port: tcp-prometheus-servicemonitor
+    scheme: http
+  selector:
+    matchLabels:
+      app.kubernetes.io/component: controller
+      app.kubernetes.io/instance: cert-manager
+      app.kubernetes.io/name: cert-manager
```

Comment:
Can you also add a bit more explaination on few of the critical user configurable fields? Like `endpoints.port`, `matchLabels`? As some of these values can be modified by the user and some cannot, this should be clear in the document.